src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -91,41 +55,122 @@ spec:
+
+```yaml
+rbac:
+  admins:
+  - customer:platform-admin        # Maps 'platform-admin' group from 'customer' connector
+  - customer:ops-team
+  editors:
+  - customer:development-team
+  viewers:
+  - customer:support-team
+```
 
-Most Giant Swarm installations use Dex as the identity provider. When using Dex, specify groups using the format `{dex-connector-id}:{group-name}`. For example:
+**Finding your connector ID:** Check your cluster's Dex configuration for the connector ID (usually `customer`).
 
-- `customer:platform-admin` - Maps the `platform-admin` group from the `customer` Dex connector to Grafana Admin role
-- `customer:development-team` - Maps the `development-team` group from the `customer` Dex connector to Grafana Editor role
+**Required fields:** Only `admins` is required; `editors` and `viewers` are optional.
 
-You can find your Dex connector ID in your cluster's Dex configuration. Only the `admins` field is mandatory in the RBAC section.
+## What happens when you create an organization
 
-For more details about organization mapping, see the official Grafana [OAuth documentation](https://grafana.com/docs/grafana/next/setup-grafana/configure-security/configure-authentication/generic-oauth/#configure-role-mapping).
+Creating a `GrafanaOrganization` resource automatically provisions:
 
-## Tenant governance
+1. **New Grafana organization** with your specified display name
+2. **Tenant-scoped datasources** for Loki, Mimir, and Alertmanager
+3. **User role assignments** based on your RBAC configuration
+4. **Tenant validation** so only defined tenants can receive data (see [tenant governance]({{< relref "/tutorials/observability/multi-tenancy#tenant-governance" >}}))
 
-The `tenants` field in your `GrafanaOrganization` serves two purposes:
+## Verification steps
 
-1. **Access Control**: Grants the organization access to data from the specified tenants
-2. **Tenant Governance**: Acts as a safeguard mechanism in the observability platform, preventing collectors from pushing to an invalid tenant.
+After creating your organization:
 
-### How tenant governance works
+1. **Check organization status:**
 
-The observability platform implements tenant governance by only accepting data for tenants that belong to **at least one** `GrafanaOrganization` resource. This means:
+   ```bash
+   kubectl get grafanaorganization myonlineshop -o yaml
+   ```
 
-- **Valid tenants**: Only tenants referenced in existing `GrafanaOrganization` resources can receive data
-- **Data rejection**: The system drops data sent to tenants not listed in any `GrafanaOrganization` resource
-- **Shared access**: Multiple organizations can reference the same tenant, allowing shared access to the same dataset
+2. **Log in to Grafana** and verify:
+   - New organization appears in the organization dropdown
+   - Datasources are configured for your tenants
```

Comment:
wondering how to check "datasources are configured for your tenants".
I was going for a simple switch to active voice, but you can't verify anything like that:
```
The `Data sources" section shows connections for each of your tenants
```

So, so far my best suggestion is:
```suggestion
   - The organization dropdown menu shows all expected organization
   - You can explore and query data for each of your tenants
```

Reply:
Oh, the next section describes how you "test data access". So I guess that's how you can explore you have data for your tenants, right? Or is there an easy step before?